src/parsing/parser.dyp

```diff
@@ -167,11 +168,16 @@ prim1 :
   | ADD1 { Add1 }
   | SUB1 { Sub1 }
   | NOT { Not }
+  | BOX { Box }
+  | UNBOX { Unbox }
   | ISBOOL { IsBool }
   | ISNUM { IsNum }
   | ISTUPLE { IsTuple }
 
-binop_expr:
+expr :
```

Comment:
I made this alias because it's weird to say that lots of things take a `binop_expr`, when what you really want to say is that they take any `expr`.